annotation-processor/src/main/java/org/hibernate/validator/ap/ConstraintAnnotationVisitor.java

```diff
@@ -112,6 +112,23 @@ public Void visitVariableAsField(VariableElement annotatedField, List<Annotation
 	/**
 	 * <p>
 	 * Checks whether the given annotations are correctly specified at the given
+	 * method parameter. The following checks are performed:
+	 * </p>
+	 * <ul>
+	 * <li>
+	 * Overridden method parameters cannot have wider range of constraints.
```

Comment:
OK, so everything looks good apart from this comment here. With this patch, we actually check the constraints validity on parameters. I think the comment is meant for the other patch. Could you fix it? Then I'll merge this one. Thanks!